label_studio/ml_models/models.py

```diff
@@ -75,3 +76,52 @@ class ThirdPartyModelVersion(ModelVersion):
 
     def has_permission(self, user):
         return user.active_organization == self.organization
+
+
+class ModelRun(models.Model):
+    class ProjectSubset(models.TextChoices):
+        ALL = 'All', _('All')
```

Comment:
Is it pretty common to `gettext_lazy as _`? Seems a little hard to interpret, but if it's standard then that's fine I guess.

Reply:
It is yes. Weird? yes. Used everywhere? yes